library/src/scala/tasty/Reflection.scala

```diff
@@ -2997,7 +2997,7 @@ class Reflection(private[scala] val internal: CompilerInterface) { self =>
   }
 
   /** Bind the `rhs` to a `val` and use it in `body` */
-  def let(rhs: Term)(body: Ident => Term): Term = {
+  def let(rhs: Term)(body: Ident => Term)(given ctx: Context): Term = {
```

Comment:
Change  to `using`?

Reply:
We still need to update the reference compiler to be able to use `using`.